project_task_personal_stage_auto_fold/models/project_task.py

```diff
@@ -0,0 +1,42 @@
+from odoo import models
+
+
+class Task(models.Model):
+    _inherit = "project.task"
+
+    def write(self, vals):
+        result = super().write(vals)
+        if "stage_id" in vals and vals.get("stage_id"):
+            if self.env["project.task.type"].browse(vals.get("stage_id")).is_closed:
```

Comment:
`is_closed` field on Tasks means the task is folded, not on a closing stage. 
I want to move the task to a "done" personal stage when the task is really done.

This is why I'm checking the field `is_closed` on the stage, not on the task.

The field on the task should may be called `is_folded` to avoid confusion.

Reply:
But AFAICS, `project.task.type` doesn't have `is_closed` field, right? See https://github.com/odoo/odoo/blob/ad171e10f0f338c6fb81d98434e26a0860e801bc/addons/project/models/project.py#L63-L105

So, in Odoo 15, folded and closed stages are synonyms. At least, that's what I understand... Am I missing something?

Reply:
On current 15.0, there's a `is_closed` field on project.task.type:

https://github.com/odoo/odoo/blob/2ba7e054b75c7408588baca2179715b0e3af8160/addons/project/models/project.py#L101

We are talking about removing `project_task_stage_closed` OCA module due to this.

Reply:
Ah! I must have been looking v14 code all this time 🤦🏼‍♂️

Reply:
Ah no, I was looking on master. So, indeed, in master, they remove the "closed" concept and make folded = closed.

https://github.com/odoo/odoo/commit/f4c054e4731270a5103ec1d5a97c2789aef977e3#diff-dcdc6e8a2e65aff0582c53d72c3890c1b56725a8b9fcf7abbb0378097b5999afL965-R962

Reply:
Oh, that's ugly. For me it's not the same...